workflow/executor/resource.go

```diff
@@ -114,8 +118,27 @@ func (g gjsonLabels) Get(label string) string {
 	return gjson.GetBytes(g.json, label).String()
 }
 
+func (we *WorkflowExecutor) signalMonitoring() {
+	log.Infof("Starting signal monitoring")
+	sigs := make(chan os.Signal, 1)
+
+	signal.Notify(sigs, os_specific.GetOsSignal())
+	go func() {
+		for {
+			<-sigs
+			log.Infof("Received update signal.")
```

Comment:
update message to say what signal did we receive and what are we doing to do as a result?